ds3-autogen-java/src/test/java/com/spectralogic/ds3autogen/java/generators/requestmodels/BaseRequestGenerator_Test.java

```diff
@@ -486,46 +494,52 @@ public void splitUuidConstructor_NoUuid_Test() {
         final ImmutableList<Arguments> args = ImmutableList.of(
                 new Arguments("String", "StringArg"),
                 new Arguments("int", "IntArg"));
-        final RequestConstructor constructor = new RequestConstructor(args, args, argsToQueryParams(args), "");
+
+        final ImmutableList<ConstructorParam> params = ImmutableList.of(
+                new SimpleConstructorParam("StringArg", "String"),
+                new SimpleConstructorParam("IntArg", "int"));
+
+        final RequestConstructor constructor = new RequestConstructor(params, args, argsToQueryParams(args), ImmutableList.of(), "");
 
         final ImmutableList<RequestConstructor> result = splitUuidConstructor(constructor);
         assertThat(result.size(), is(1));
     }
 
+
+
     @Test
     public void splitUuidConstructor_Test() {
-        final ImmutableList<Arguments> args = ImmutableList.of(
-                new Arguments("String", "StringArg"),
-                new Arguments("UUID", "UuidArg"),
-                new Arguments("int", "IntArg"));
-        final RequestConstructor constructor = new RequestConstructor(args, args, argsToQueryParams(args), "");
+
+        final RequestConstructor constructor = new RequestConstructor(SIMPLE_PARAMS, SIMPLE_ARGS, argsToQueryParams(SIMPLE_ARGS), ImmutableList.of(), "");
 
         final ImmutableList<RequestConstructor> result = splitUuidConstructor(constructor);
         assertThat(result.size(), is(2));
-        assertThat(getAllArgumentTypes(result.get(0).getParameters()), hasItem("UUID"));
+
+        assertTrue(result.get(0).getParameters().stream()
```

Comment:
Consider using AssertJ for these kinds of asserts, and all other asserts (but we can do that later.)